components/CheckDocumentation/connections/BlockingQuery.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+
+import SQL from "../../SQL";
+
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const BlockingQueryTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  return (
+    <>
+      <p>
+        Detects queries currently blocking equal or more than the specified
+        threshold of <code>{config.settings["blocked_count"]}</code> queries.
+        When any blocked query is running for longer than the specified
+        threshold of <code>{config.settings["warning_blocked_age_secs"]}</code>{" "}
```

Comment:
I think same here, I'm leaning toward dropping "the specified threshold of". Thoughts?